docs/welcome.mdx

```diff
@@ -38,140 +38,94 @@ improved hourly.
 
-Ory is the largest open source ecosystem in the area of authentication, authorization, access control, and zero trust networking
-in the world. Ory is not another company "greenwashing" with open source by publishing SDKs under open-source licenses. Instead,
-all of our core systems are available as Apache 2.0 licensed software without enterprise or open core models.
+Ory Network incorporates the open-source [Ory Hydra OAuth2 & OpenID Server](https://www.ory.sh/hydra) and offers:
 
-Head over to the [Ory Open Source Overview](./open-source.mdx) for an introduction to the different projects.
+- **Fully featured OAuth2 & [OpenID Certified](https://openid.net/developers/certified/)® OIDC Provider**
 
-### Is open source taking a back seat?
-
-**No!** The Ory Network is our commercial offering with which we generate revenue to build more open source and more features in
-Ory Network. Our value proposition and vision for The Ory Network is to incorporate and advance the open-source ecosystem and add
-additional services which aren't possible to open source (for example analytics, SLAs, SRE, low-latency).
-
-The Ory Network and our open-source software are interlinked. While we've made some modifications to the open source to make it
-work better in our multi-tenant environment and operational (SRE) model, we're using the same code base, APIs, features, and
-configurations for The Ory Network.
-
-In the future, we will increase our open-source footprint, make the projects easier to modify, use, access, and distribute. The
-only impact The Ory Network will have is that it will accelerate development and enable us to spend more resources on the open
-source!
-
-## New to Ory ?
-
-- [Ory Projects Overview.](/ecosystem/projects.mdx) What is Ory ? Learn about the open-source projects at the heart of Ory:
-- [Ory Community Overview.](/ecosystem/community.md) A place where developers share and build: Learn more about the Ory Community:
-- [Ory Kratos Quickstart.](https://www.ory.sh/docs/kratos/quickstart) User login and registration for web, mobile, native, or
-  single-page applications.
-- [Ory Keto Quickstart.](https://www.ory.sh/docs/keto/quickstart) Complex consistent permission and authorization management.
-- [Ory Hydra Quickstart.](https://www.ory.sh/docs/hydra/5min-tutorial) Authenticate third-party users and secure access to your
-  applications and APIs.
-- [Ory Oathkeeper Quickstart.](https://www.ory.sh/docs/oathkeeper/configure-deploy) Authenticate, authorize and mutate any
-  incoming network traffic, with a Zero Trust security model.
-
-Don't want to deal with Docker, Kubernetes, and securing your API?
+### Ory Console
 
-- [Sign up](https://console.ory.sh/registration) for a free Developer account in The Ory Network.
+Ory Console is the management UI of Ory Network. It offers capabilities such as showing and searching for identities, managing
+user sessions, and gaining valuable insights in the form of dashboards and customization.
```

Comment:
Customization isn't a "valuable insight", is it?
As for dashboards - these aren't in the Console yet.

Reply:
Agreed. 

I would say in that sentence we dont need more than 

"Ory Console is the management UI of Ory Network."

There is too many features being added all the time that this description would not be outdated very quickly. 
IMO the above keeps it simple and accurate. 
You manage Ory Network through the console, thats about it, no need to write a marketing letter here.